Cargo.toml

```diff
@@ -12,6 +12,7 @@ common-build = { path = "common-build" }
 
 [dependencies]
 cc3200-sys = { path = "cc3200-sys" }
+cfg-if = { path = "../cfg-if" }
```

Comment:
I'm not sure how this worked. What I did was to make a local copy of cg_if and added a #[no_std] since the only thing it needs std for is the tests.